data/mods/No_Explosive_Zombies/modinfo.json

```diff
@@ -0,0 +1,17 @@
+[
+{
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
+    "ident": "No_Explosive_Zombies",
+    "name": "No Explosive Zombies",
+    "description": "Removes all explosion-based zombies from the game.",
```

Comment:
What about boomer and bloated types, or are their explosions not deadly enough for this list?